cfme/tests/infrastructure/test_config_management.py

```diff
@@ -160,7 +160,11 @@ def test_ansible_tower_job_templates_tag(request, config_manager, tag):
 
 @pytest.mark.tier(3)
 @pytest.mark.uncollectif(lambda config_manager_obj: config_manager_obj.type != "Ansible Tower")
-def test_ansible_tower_service_dialog_creation_from_template(request, config_manager, appliance):
+@pytest.mark.parametrize('template_type', ['Job Template (Ansible Tower)',
+    'Workflow Template (Ansible Tower)'], ids=['job', 'workflow'],
+    scope='module')
```

Comment:
Q: why scope as `module`?

also optional... better to put key value pair for type and name so we can use in other test parametrization like
```
TEMPLATE_TYPE = {
    "job": "Job Template (Ansible Tower)",
    "workflow": "Workflow Template (Ansible Tower)",
}
@pytest.mark.parametrize("template_type", TEMPLATE_TYPE.values(), ids=[TEMPLATE_TYPE.keys()])
def test_foo(template_type):
    pass
```

Reply:
Done